service/conf/routes

```diff
@@ -18,6 +18,7 @@ GET  	/v2/user/read/:uid				  @controllers.usermanagement.UserController.getUser
 POST  /v1/user/getuser            @controllers.usermanagement.UserController.getUserByLoginId()
 GET  /v1/user/get/:idType/:id           @controllers.usermanagement.UserController.getUserByKey(idType:String,id:String)
 POST  /v1/user/search					    @controllers.usermanagement.UserController.searchUser()
+GET /v1/user/private/read/:externalId  @controllers.usermanagement.UserController.getUserById(externalId:String)
```

Comment:
/private/v1/user/read/externalId?provider=providervalue?&idType=idTyeValue

as per latest review - above is the decided url format - please update this one accordingly. 
private is moved at the front